modules/ROOT/pages/_partials/generic-service-intros.adoc

```diff
@@ -49,7 +49,9 @@ The {keycloak-service} service allows you to add authentication and authorizatio
 = {digger-service}
 //tag::mobilecicd-service[]
 
-The {digger-service} service allows you automate the building of your native and hybrid mobile app for both Android and iOS.
+The {digger-service} service provides a system to bring together the product of developers work into a 
```

Comment:
not what was agreed with PM/Eng, maybe put it somewhere else?

Reply:
Hey @finp where can I see what we did agree with them?

Reply:
Is that the original text?